src/util.js

```diff
@@ -106,3 +107,127 @@ export function format_memory_and_limit(usage, limit) {
         return _("");
     }
 }
+
+// TODO
+export function container_stop(container, timeout) {
+}
+
+export function updateContainers() {
+    let newContainers = [];
+    let newContainersStats = [];
+    return new Promise((resolve, reject) => {
+        varlinkCall(PODMAN, "io.podman.ListContainers")
+                .then(reply => {
+                    let newContainersMeta = reply.containers;
+                    if (!newContainersMeta) {
+                        resolve({newContainers: newContainers, newContainersStats: newContainersStats});
+                        return;
+                    }
+                    const len = newContainersMeta.length;
+                    let inspectRet = [];
+                    newContainersMeta.map((container) => {
+                        let proEle = new Promise((resolve, reject) => {
+                            varlinkCall(PODMAN, "io.podman.InspectContainer", {name: container.id})
+                                    .then(reply => {
+                                        resolve(JSON.parse(reply.container));
+                                    })
+                                    .catch(ex => {
+                                        reject(new Error("Failed to do InspectContainer call:", ex, JSON.stringify(ex)));
+                                    });
+                        });
+                        inspectRet.push(proEle);
+                    });
+                    Promise.all(inspectRet)
+                            .then((inspectRet) => {
+                                let runEle = newContainersMeta.filter((ele) => {
+                                    return ele.status === "running";
+                                });
+                                inspectRet.map((inspectRet) => {
+                                    newContainers.push(inspectRet);
+                                    if (runEle.length === 0 && newContainers.length === len) {
+                                        resolve({newContainers: newContainers, newContainersStats: newContainersStats});
+                                    }
+                                });
+                            })
+                            .catch(ex => {
+                                console.error("Failed to do InspectContainer call:", ex, JSON.stringify(ex));
+                            });
+                    let crtStatsRet = [];
```

Comment:
is this mean to be `ctrStatsRet`? Might be better to spell out the names, to avoid having to guess what acronyms mean. :)